.rubocop.yml

```diff
@@ -34,9 +34,7 @@ Metrics/ClassLength:
   Enabled: true
 
 Metrics/ModuleLength:
-  # TODO: bring this down to 100
-  Max: 150
-  Enabled: true
+  Enabled: false
```

Comment:
This change seems to only be for the `spec/services/taxonomy/taxonomy_query_spec.rb` file. Perhaps we could describe the class in the spec as follows:

```diff
- module Taxonomy
-   RSpec.describe TaxonomyQuery do
+ RSpec.describe Taxonomy::TaxonomyQuery do
```

Reply:
fixed